artiq/dashboard/experiments.py

```diff
@@ -267,8 +274,9 @@ def __init__(self, manager, expurl):
         if scheduling["due_date"] is None:
             datetime.setDate(QtCore.QDate.currentDate())
         else:
-            datetime.setDateTime(QtCore.QDateTime.fromMSecsSinceEpoch(
-                scheduling["due_date"]*1000))
+            # Casting to int to prevent type error leaks
```

Comment:
```
Traceback (most recent call last):
  File "/nix/store/8qnzzl4zysdzz616yi96gmzcbnzhragm-python3-3.10.11-env/lib/python3.10/site-packages/artiq/dashboard/explorer.py", line 288, in expname_action
    action("repo:" + expname)
  File "/nix/store/8qnzzl4zysdzz616yi96gmzcbnzhragm-python3-3.10.11-env/lib/python3.10/site-packages/artiq/dashboard/experiments.py", line 725, in open_experiment
    dock = _ExperimentDock(self, expurl)
  File "/nix/store/8qnzzl4zysdzz616yi96gmzcbnzhragm-python3-3.10.11-env/lib/python3.10/site-packages/artiq/dashboard/experiments.py", line 282, in __init__
    datetime.setDateTime(QtCore.QDateTime.fromMSecsSinceEpoch(
TypeError: arguments did not match any overloaded call:
  fromMSecsSinceEpoch(msecs: int): argument 1 has unexpected type 'float'
  fromMSecsSinceEpoch(msecs: int, spec: Qt.TimeSpec, offsetSeconds: int = 0): argument 1 has unexpected type 'float'
  fromMSecsSinceEpoch(msecs: int, timeZone: QTimeZone): argument 1 has unexpected type 'float'
./dashboard.sh: line 3: 96111 Aborted                 (core dumped) nix shell --command artiq_dashboard
```
This was the error message. It can occur in the following sequence:
1. Current experiment window is open, applet tries to modify entries 
2. Entry values changed
3. Then try to modify the values through entries widget and clicking Submit

Casting to int temporarily solves the issue. Will investigate more to find out the root cause.

Reply:
Yep, somebody just found this here as well.

Reply:
Bug fix transferred to PR #2136